gokrazy.go

```diff
@@ -28,6 +29,8 @@ var (
 	buildTimestamp = "uninitialized"
 	httpPassword   string
 	hostname       string
+	tlsConfig      *tls.Config = nil
```

Comment:
No need for the `= nil`, variables are initialized to their zero value by default.